### DISCUSSION TITLE
Suggestions for new features

### DISCUSSION THREAD

**Discussion by Chrstn67** [facebook/docusaurus#10304] [Ideas]
I discovered Docusaurus very recently. I love it!
I'm a web developer and I'm used to giving conferences. 
One feature I'd love to see implemented is the ability to create mind maps. 
I'd love to be able to create my own mind maps, then have the choice of displaying them in “mindmap” mode or in markdown.
The reverse functionality would also be exceptional: our markdown is converted into a mindmap.

Thank for your job

**Comment by slorber** [comment]
Thanks

We already have support for mindmaps thanks to our Mermaid integration:
https://docusaurus.io/blog/releases/3.0#mermaid-diagrams
However, you can use any React component providing mindmap support as well.
